### PR TITLE
chapter 4

### PR DESCRIPTION
**Pull request by leanhviet**

### CODE REVIEW

**Comment by leanhviet** [comment]
t

**Comment by leanhviet** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (dothidiemthao) [inline app/helpers/application_helper.rb]: i18n "Ruby on Rails Tutorial Sample App"

**Comment by dothidiemthao** [comment]
commented

**Comment by leanhviet** [comment]
tttttt

**Comment by leanhviet** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**